Cargo.lock

```diff
@@ -926,7 +926,7 @@ dependencies = [
 [[package]]
 name = "cosmic-settings-config"
 version = "0.1.0"
-source = "git+https://github.com/pop-os/cosmic-settings-daemon#362c77f9faaeb7f1b9e4aa79a7d5588001f04874"
+source = "git+https://github.com/pop-os/cosmic-settings-daemon#1ed68808e85ce681da882446ec572d44c68a6866"
```

Comment:
> Did the cargo update, some of the version moves in the diff looked confusing, such as libloading 0.8.5 -> 0.7.4. Pushed it up in case it is correct.

Yeah it is weird, I am seeing the same changes locally running that command, but libcosmic definitely doesn't need to be updated for this. (And I very much would prefer it to stay on the current version given I would likely need to re-test a bunch of stuff, if we updated it.)

From my testing locally you can drop every `Cargo.lock` change, except this line and it will work.

Reply:
Alright, that makes sense. forced that up.